src/renderer/globalVars.js

```diff
@@ -0,0 +1,26 @@
+import * as THREE from 'three'
+import { GLTFLoader } from 'three/examples/jsm/loaders/GLTFLoader.js'
+import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls.js'
+import * as postprocessing from 'postprocessing'
+import glslify from 'glslify'
+import TWEEN from '@tweenjs/tween.js'
+
+window.HEDRON = {
+  // Third party dependencies exposed globally for sketch development
+  dependencies: {
+    // Declaring THREE as a global var, so that sketches can use the same instance of three.js as Hedron does
+    // This keeps the library versions matched and also prevents strange things from happening when the library
+    // code is being read from different sources
+    THREE: {
+      ...THREE,
+      // For convenience, also requiring some common three extras
+      GLTFLoader,
```

Comment:
Not needed for this pr, but it might be handy to have as many loaders as we can get our hands on, will help lead way to an asset importing/library type thing

Reply:
Definitely